app/helpers/application_html_formatters_helper.rb

```diff
@@ -11,15 +11,43 @@ module ApplicationHTMLFormattersHelper
                                          HTML::Pipeline::RougeFilter
                                        ], DefaultPipelineOptions)
 
+  VIDEO_WHITELIST_TRANSFORMER = lambda do |env|
+    node      = env[:node]
+    node_name = env[:node_name]
+
+    return if env[:is_whitelisted] || !node.element?
+    return unless node_name == 'iframe'
+
+    return unless node['src'] =~ %r{
```

Comment:
Just wondering, is there another way to write this in a more extensible way (without sacrificing performance)? 

Actually I have some youtube URLs to be whitelisted in a URL concern for the video model as well, not sure if we can combine as well. We can go ahead with this first and refactor that later on.